pkg/gui/custom_commands.go

```diff
@@ -177,17 +200,18 @@ func (gui *Gui) menuPromptFromCommand(prompt config.CustomCommandPrompt, promptR
 	}
 
 	// Need to make a menu out of what the cmd has displayed
-	candidates, err := gui.GenerateMenuCandidates(message, filter, prompt.Format)
+	candidates, err := gui.GenerateMenuCandidates(message, filter, prompt.ValueFormat, prompt.LabelFormat)
 	if err != nil {
 		return gui.surfaceError(err)
 	}
 
 	menuItems := make([]*menuItem, len(candidates))
 	for i := range candidates {
 		menuItems[i] = &menuItem{
-			displayStrings: []string{candidates[i]},
+			// Put in candidate and its description
+			displayStrings: []string{candidates[i].value, style.FgYellow.Sprint(candidates[i].label)},
```

Comment:
Can we change this to only show the label?
This way the user can fully define how the output looks on screen and so it not restricted to value always first then the label we can (in a later pr) add colours functions to the template so users have the full freedom to whatever they want to show on screen.
```suggestion
			displayStrings: []string{candidates[i].label},
```